src/pages/index.fr-fr.js

```diff
@@ -0,0 +1,198 @@
+      des modules rend la tâche contraignante, et revient souvent à empiler les
+      couches d'abstraction. eBPF est une technologie révolutionnaire
+      permettant d'exécuter des programmes au sein du noyau Linux, dans un
+      environnement confiné, sans changer le code source du noyau ni charger de
+      modules.
+    </p>
+    <p>
+      Grâce à cette capacité à programmer le noyau, les composants logiciels
+      peuvent s'appuyer sur le système existant. Ils viennent le perfectionner
+      et l'enrichir en fonctionnalités, en évitant de rajouter des couches
+      d'abstraction, mais sans compromettre l'efficacité ou la sureté du
+      système.
+    </p>
+    <img src={require("../assets/go.png")} />
+    <p>
+      eBPF a conduit au développement d'une génération entièrement nouvelle
+      d'applications, capables de reprogrammer le noyau Linux, et même
+      de mettre en œuvre une logique commune à différents sous-systèmes du
+      noyau, jusqu'ici décorrélés les uns des autres.
+    </p>
+  </div>
+);
+
+const Sections = () => (
+  <div className="main-sections" style={{ marginTop: "30px" }}>
+    <div className="main-sections-left">
+      <Section
+        title="Sécurité"
+        icon={require("../assets/intro_security.png")}
+        text={securityText}
+      />
+      <Section
+        title="Réseau"
+        icon={require("../assets/intro_networking.png")}
+        text={networkingText}
+      />
+    </div>
+    <div className="main-sections-right">
+      <Section
+        title="Traçage et profilage"
+        icon={require("../assets/intro_tracing.png")}
+        text={tracingText}
+      />
+      <Section
+        title="Observabilité et supervision"
+        icon={require("../assets/intro_observability.png")}
+        text={monitoringText}
+      />
+    </div>
+  </div>
+);
+
+const Outro = () => (
+  <div className="intro">
+    <p>
+      Pour en apprendre plus sur eBPF et ses applications :
+      <table width="100%">
+        <tr>
+          <td>
+            <ul>
+              <li>
+                <a href="/what-is-ebpf">Comprendre eBPF</a>
+              </li>
+              <li>
+                <a href="/projects">Liste de projets basés sur eBPF</a>
+              </li>
+            </ul>
+          </td>
+          <td>
+            <ul>
+              <li>
+                <a href="/slack">
+                  Rejoindre la communauté #ebpf sur Slack
+                </a>
+              </li>
+              <li>
+                <a href="/contribute">Comment contribuer</a>
+              </li>
+            </ul>
+          </td>
+        </tr>
+      </table>
+    </p>
+  </div>
+);
+
+const IndexPage = ({ data }) => {
+  return <Layout>
+    <div className="page-wrapper page-index">
+      <IndexHelmet />
+      <MainTitle />
+      <Buttons />
+      <Intro />
+      <Sections />
+      <Videos title="Quelques présentations sur eBPF" />
```

Comment:
```suggestion
      <Videos title="Sélection de présentations sur eBPF" />
```

Reply:
I hesitated too. Having “Sélection” in several titles felt a bit heavy. Will go with “Découvrir eBPF en vidéo”.